components/multiselect/multiselect_list.jsx

```diff
@@ -165,8 +168,8 @@ export default class MultiSelectList extends React.Component {
         } else {
             renderer = this.defaultAriaLabelRenderer;
         }
-        const selectedOption = options.filter((o, i) => this.state.selected === i);
-        const ariaLabel = renderer(selectedOption[0] || {});
+        const selectedOption = options[this.state.selected];
+        const ariaLabel = renderer(selectedOption);
```

Comment:
I was actually thinking something like
```javascript
const ariaLabel = selectedOption ? renderer(selectedOption) : '';
```
but this way works as well

Reply:
This way the consumer can determine whatever blank message they want! :tada: